stdlib/contextlib.pyi

```diff
@@ -36,7 +36,7 @@ class _GeneratorContextManager(AbstractContextManager[_T_co]):
     def __call__(self, func: _F) -> _F: ...
 
 # type ignore to deal with incomplete ParamSpec support in mypy
-def contextmanager(func: Callable[_P, Iterator[_T]]) -> Callable[_P, _GeneratorContextManager[_T]]: ...  # type: ignore[misc]
```

Comment:
`_GeneratorContextManager` is also  used in `stubs/decorator/decorator.pyi`.

- Should we change `decorator.pyi` too?
- If we change both, should we delete `_GeneratorContextManager`?

Reply:
Apparently `decorator`'s documentation explains what the `__call__` method is supposed to do: https://github.com/micheles/decorator/blob/master/docs/documentation.md#contextmanager

Maybe we should just keep it, because it is a documented feature.

Reply:
Another observation: the implementation of `decorator` just does `from contextlib import _GeneratorContextManager`, and uses it as a base class. So their solution to #6520 (or rather, a similar problem at runtime) is to just import it and not worry about it.